java/tv/sage/ModuleGroup.java

```diff
@@ -293,27 +293,32 @@ else if (cryptoKeyObj instanceof Object[])
         }
       }
       file.createNewFile();
-      sage.FastRandomFile widgetDBout = new sage.FasterRandomFile(file, cryptoKey == null ? "rwd" : "rwdc", sage.Sage.I18N_CHARSET);
-      if (cryptoKey != null) widgetDBout.setCrypto(cryptoKey);
+      sage.io.SageDataFile widgetDBout;
+
+      if (cryptoKey == null)
```

Comment:
Leave the code the way it is, since that's likely correct. Feel free to update the comment. :)